### DISCUSSION TITLE
error after a while running

### DISCUSSION THREAD

**Discussion by Wade-BAV** [victornpb/undiscord#312] [1. Questions & Answers] [answered]
hey i tried out the old copy paste and the new version but i get always that error now
"Ended because API returned an empty page"

is there a  way to make it running again?

**Comment by Toby222** [comment]
That means search was empty. Might be because of API outage, if it persists for longer, try other channels and such.

**Reply by Wade-BAV** [reply]
> That means search was empty. Might be because of API outage, if it persists for longer, try other channels and such.

ah no as i said its running a while then then it stopped

**Comment by victornpb** [comment, marked as answer]
The popup version was discontinued as it was lagging behind. You can still copy/paste Undiscord in the console if you prefer to run it that way, it should work perfectly fine.
